### ISSUE TITLE
type.indexOf is not a function

### ISSUE THREAD

**Issue by jonathanong** [jshttp/type-is#19]
seeing this error:

```
EXPRESS ERROR:  TypeError: type.indexOf is not a function 
    at normalize (/app/node_modules/express/node_modules/type-is/index.js:174:15) 
    at typeis (/app/node_modules/express/node_modules/type-is/index.js:70:19) 
    at typeofrequest (/app/node_modules/express/node_modules/type-is/index.js:142:10) 
    at IncomingMessage.is (/app/node_modules/express/lib/request.js:270:10) 
    at exports.upload (/app/app/controllers/api/videos.js:54:14) 
    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5) 
    at next (/app/node_modules/express/lib/router/route.js:131:13) 
    at exports.user.exists (/app/app/middleware/authorization.js:30:3) 
    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5) 
    at next (/app/node_modules/express/lib/router/route.js:131:13) 
    at Route.dispatch (/app/node_modules/express/lib/router/route.js:112:3) 
    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5) 
    at /app/node_modules/express/lib/router/index.js:277:22 
    at Function.process_params (/app/node_modules/express/lib/router/index.js:330:12) 
    at next (/app/node_modules/express/lib/router/index.js:271:10) 
    at /app/node_modules/express/lib/application.js:232:9 
    at /app/node_modules/express/lib/router/index.js:615:15 
    at next (/app/node_modules/express/lib/router/index.js:256:14) 
    at compression (/app/node_modules/compression/index.js:205:5) 
    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5) 
    at trim_prefix (/app/node_modules/express/lib/router/index.js:312:13) 
    at /app/node_modules/express/lib/router/index.js:280:7 
    at Function.process_params (/app/node_modules/express/lib/router/index.js:330:12) 
    at next (/app/node_modules/express/lib/router/index.js:271:10) 
    at expressInit (/app/node_modules/express/lib/middleware/init.js:33:5) 
    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5) 
    at trim_prefix (/app/node_modules/express/lib/router/index.js:312:13) 
    at /app/node_modules/express/lib/router/index.js:280:7 
    at Function.process_params (/app/node_modules/express/lib/router/index.js:330:12) 
    at next (/app/node_modules/express/lib/router/index.js:271:10) 
    at query (/app/node_modules/express/lib/middleware/query.js:49:5) 
    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5) 
    at trim_prefix (/app/node_modules/express/lib/router/index.js:312:13) 
    at /app/node_modules/express/lib/router/index.js:280:7 
```

going through the source code but can't find where this would error. any ideas?

**Comment by jonathanong** [comment]
i guess you could just do an early return on a falsey `type` in `normalize()`, but i can't figure out how a falsey value would get there in the first place...

**Comment by dougwilson** [comment]
It comes in from the list. Example: `is(req, 'json', undefined, false, 'hello')` I believe will generate the error.

**Comment by dougwilson** [comment]
Published to npm as 1.6.7
